From a1dfaf1e6fd20545e1a3bae784bdaaf742a88a65 Mon Sep 17 00:00:00 2001
From: Sebastian Krzyszkowiak <dos@dosowisko.net>
Date: Thu, 30 Aug 2018 03:09:51 +0200
Subject: [PATCH 2/5] Don't include GLES1 headers when GLES2 is active

GLES1 and GLES2 can't be included at the same time in Maemo
---
 src/video/SDL_video.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/video/SDL_video.c b/src/video/SDL_video.c
index 336fdaa50..4b43449cf 100644
--- a/src/video/SDL_video.c
+++ b/src/video/SDL_video.c
@@ -37,7 +37,7 @@
 #include "SDL_opengl.h"
 #endif /* SDL_VIDEO_OPENGL */
 
-#if SDL_VIDEO_OPENGL_ES
+#if SDL_VIDEO_OPENGL_ES && !SDL_VIDEO_OPENGL_ES2
 #include "SDL_opengles.h"
 #endif /* SDL_VIDEO_OPENGL_ES */
 
-- 
2.20.1

